impeller/entity/entity_pass.cc

```diff
@@ -247,7 +247,8 @@ bool EntityPass::Render(ContentContext& renderer,
                         const RenderTarget& render_target) const {
   auto root_render_target = render_target;
 
-  if (root_render_target.GetColorAttachments().empty()) {
+  if (root_render_target.GetColorAttachments().find(0u) ==
```

Comment:
The linux embedder API was sending a bad value through here, so the map wasn't empty. It doesn't matter that its empty though, it matters that 0u exists because that is what we use below.

Reply:
It might be worth extracting a named constant for this to avoid the magic number.